packages/aragon-cli/src/commands/ipfs.js

```diff
@@ -1,9 +1,17 @@
-const startCommand = require('./ipfs_cmds/start')
+import * as commands from './ipfs_cmds'
 
-exports.builder = function(yargs) {
-  return yargs.commandDir('ipfs_cmds')
-}
+export const builder = yargs =>
```

Comment:
Should we apply this pattern to all other subcommands? I like this more explicit approach than using `yargs.commandDir`, but is there a particular reason for it?

Reply:
Yes, because it affects the order `--help` is generated and I think we can add them in the order an user will most likely learn/use them.

It's a very small detail though so I wouldn't worry about changing the other commands, unless you're within that file or something :man_shrugging:

Reply:
Sound good! Let's add this as a todo task for latter down the refactor road